content/guidelines/reviewing.en.md

```diff
@@ -122,6 +122,18 @@ As you did not push to the source branch but instead created a copy of that bran
 
 Once the pull request has been merged, delete the copy you made of the branch with `git push origin -d add_<service_name>_<terms_type>`.
 
+### Debugging failing Tests
+
+When tests fail, you can follow these steps to diagnose and address the issue:
+
+1. Begin by analyzing any error messages or warnings provided by the test output. These messages can often provide meaningful information to identify the source of the problem. Pay attention to specific issues such as schema validation errors, inaccessible web locations or inaccessible content selection.
```

Comment:
I feel that the main problem here is that newbies might not know what the “test output” is, let alone how to access it…

Reply:
Maybe this content is misplaced, but here we are in a document for reviewer "Reviewing contributions", not really for newbies